code/solutions/Flint.build0/models/Flint.build.mps

```diff
@@ -218,6 +218,7 @@
         <child id="5253498789149547704" name="dependencies" index="3bR37C" />
       </concept>
       <concept id="5253498789149585690" name="jetbrains.mps.build.mps.structure.BuildMps_ModuleDependencyOnModule" flags="ng" index="3bR9La">
+        <property id="5253498789149547713" name="reexport" index="3bR36h" />
```

Comment:
I take it this file is something that needs to be committed, or not? (It does have "build" in the path, just checking:))

Reply:
Yeah that contains the information on how to build the plugin / ide.

Reply:
I didn't notice any performance improvements and it's probably a good idea to check if you can still import json and open ComplianceByDesign.

Reply:
So, as it turns out I can import the models, but I can't then open them, any thoughts?

Reply:
What exactly do you mean by can't open them?

Reply:
I mean, when I click on them in the sidebar the nice view does not open. I can right click and get a text-json view, but that's it

Reply:
Thats a new one. I can't reproduce this. Is there a stack trace?

Reply:
GitHub is being weird, so I slacked you what I get error-wise

Reply:
When running/building with Java 11 instead of 14 it suddenly works:) CbD works as well. Feel free to merge!